addons/addons.json

```diff
@@ -135,14 +134,12 @@
   "rename-broadcasts",
   "middle-click-popup",
   "number-pad",
-  "necropost-highlighter",
   "copy-reporter",
   "share-through-mystuff",
-  "faster-project-loading",
   "auto-follow-topics",
   "featured-dangos",
-  "remember-collapsed-categories",
```

Comment:
I think you accidentally removed some addons from the list. Could you add them back?